Tests/Unit/ConvertFrom-MemoryStreamToSecureString.Tests.ps1

```diff
@@ -0,0 +1,27 @@
+﻿# Dot Source Function
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path).Replace('.Tests.', '.')
+Set-Location $here
+. "..\$sut"
+
+Describe 'ConvertFrom-MemoryStreamToSecureString' {
```

Comment:
I think we should add a test to validate the secure string contains the expected values.
Perhaps create a PSCredential using it, and read it back to confirm?